utils/wasm-builder/Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "gear-wasm-builder"
-version = "0.1.1"
+version = "0.2.0"
```

```suggestion
version = "0.1.2"
```
I don't think this change requires major version bumping.

> Initial development releases starting with "0.y.z" can treat changes in "y" as a major release, and "z" as a minor release.

https://doc.rust-lang.org/cargo/reference/semver.html

---

I was doubting about whether it is breaking changes or not, but you are right, it is not quite breaking.